jest.config.cjs

```diff
@@ -1,8 +1,8 @@
 // https://github.com/ai/nanoid/issues/363#issuecomment-1140906651
-const esModulesToIgnore = ['nanoid'].join('|')
+const esModulesToIgnore = ['nanoid', 'lodash-es', 'sinon'].join('|')
```

Comment:
Are these needed? We aren't using those packages (sinon and nanoid) in the package.json

Reply:
yes